.env

```diff
@@ -0,0 +1 @@
+export SOUNDCLOUD_CLIENT_ID=69e93cf2209402f6f3137a6452cf498f
```

Comment:
It already is. :wink:
I copied it from 94dd4daa7b0b0886574a127577072686f4986d70 , so it was already compromised.